pkg/storage/unified/apistore/restoptions.go

```diff
@@ -94,10 +93,16 @@ func NewRESTOptionsGetterForFile(path string,
 	return NewRESTOptionsGetterForClient(
 		resource.NewLocalResourceClient(server),
 		originalStorageConfig,
-		features,
 	), nil
 }
 
+func (r *RESTOptionsGetter) RegisterOptions(gr schema.GroupResource, opts StorageOptions) {
+	if r.options == nil {
+		r.options = make(map[string]StorageOptions)
+	}
```

Comment:
Suggestion: We might want to run this once in `NewRESTOptionsGetterForClient` instead of here every time, as this seems to be the constructor that is always called.

Reply:
good point -- it already was, so I can remove the check